prisma/migrations/20240118234302_remove_finicity_investment_transaction_categories/migration.sql

```diff
@@ -0,0 +1,79 @@
+-- AlterTable
+ALTER TABLE "investment_transaction"
+    RENAME COLUMN "category" TO "category_old";
+
+DROP VIEW IF EXISTS holdings_enriched;
+
+ALTER TABLE "investment_transaction"
+ADD COLUMN "category" "InvestmentTransactionCategory" NOT NULL GENERATED ALWAYS AS (
+  CASE
+    WHEN "plaid_type" = 'buy' THEN 'buy'::"InvestmentTransactionCategory"
+    WHEN "plaid_type" = 'sell' THEN 'sell'::"InvestmentTransactionCategory"
+    WHEN "plaid_subtype" IN ('dividend', 'qualified dividend', 'non-qualified dividend') THEN 'dividend'::"InvestmentTransactionCategory"
+    WHEN "plaid_subtype" IN ('non-resident tax', 'tax', 'tax withheld') THEN 'tax'::"InvestmentTransactionCategory"
+    WHEN "plaid_type" = 'fee' OR "plaid_subtype" IN ('account fee', 'legal fee', 'management fee', 'margin expense', 'transfer fee', 'trust fee') THEN 'fee'::"InvestmentTransactionCategory"
+    WHEN "plaid_type" = 'cash' THEN 'transfer'::"InvestmentTransactionCategory"
+    WHEN "plaid_type" = 'cancel' THEN 'cancel'::"InvestmentTransactionCategory"
```

Comment:
@tmyracle I think you forgot to add the Teller category maps here

Reply:
Teller doesn't have investment accounts so I didn't add them

Reply:
So that means we'll only have "other" once Plaid support is removed as well?

Reply:
Yep, we'll likely have to rethink the whole InvestmentTransaction data model to accommodate a world without Plaid or Finicity

Reply:
@Shpigford Any ideas on how to best proceed here?

Reply:
I _think_ InvestmentTransaction will still work fine on a high level. We likely just need to do some work to standardize transaction types and then as we build different connectors/importers, we'll then have them meet those standards.

Plaid + Finicity almost offered _too much_ detail on the different types/subtypes in a way that added a lot of unnecessary complexity, so my vote would be to rip out the Plaid/Finicity stuff and then we focus on a task like #131 for creating the new standard, which we can then build off of.

Reply:
@tmyracle Then I guess it's fine that we'll eventually (once both Finicity & Plaid are removed) only set `other` as default value here 🤷‍♂️